docs/content/envoy-performance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Performance
+kind: envoy
+weight: 100
+---
+
+This page provides some performance benchmarks that give an idea of the overhead of using the OPA-Envoy plugin.
+
+### Test Setup
```

Comment:
💭 Maybe not on this page, but in a gist or on the opa-envoy-plugin repo -- it would be nice if the exact steps used to record those numbers below would be available. (Not a blocker)